cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperatorTest.java

```diff
@@ -1451,6 +1451,60 @@ public void testCruiseControlDisabledToEnabledBehaviour(VertxTestContext context
                 }));
     }
 
+    @Test
+    public void testKrWhenCCUserTaskIsEmpty(VertxTestContext context) throws IOException, URISyntaxException {
```

Comment:
The test name is not great IMO. Should we rename to `shouldGenerateNewProposalWhenUserTasksResponseHasNoTasks`, or something similar?

Reply:
Yeah, I think that would be a better name